src/runtime/scripts/lazy-noscript.js

```diff
@@ -0,0 +1,31 @@
+const EMPTY_GIF = 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7'
+
+const supportsLazyLoading = 'loading' in HTMLImageElement.prototype
+
+if (!supportsLazyLoading) {
+  const script = document.createElement('script')
+  script.src = 'lazy-polyfill.js'
+  script.async = true
+  document.head.appendChild(script)
+}
+
+const noscripts = document.querySelectorAll('noscript')
+
+noscripts.forEach(function (el) {
+  let html = el.innerHTML.trim().replace(/&lt;/g, '<').replace(/&gt;/g, '>')
+
+  if (!html.match(/data-src/)) {
+    return
+  }
+
+  if (!supportsLazyLoading) {
+    html = html.replace(/ src="([^"]*)"/, ' src="' + EMPTY_GIF + '"')
```

Comment:
```suggestion
    html = html.replace(/ src="([^"]*)"/g, ' src="' + EMPTY_GIF + '"')
```

Reply:
We might convert `src` to `data-src` at same stage (and match by `loading=lazy`)